dask/bag/core.py

```diff
@@ -395,7 +395,7 @@ def __str__(self):
     str = property(fget=StringAccessor)
 
     def map(self, func, *args, **kwargs):
-        """Apply a function elementwise across one or more bags.
+        """Apply a function elementwise across one or more bags
```

Comment:
Why remove the periods?

Reply:
Tried to make it consistent with the other docs within the file. Either period for all or for none (and most of them did not have periods).

Reply:
Dask follows the numpydoc standard, which uses periods for the short description (https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#sections). This is consistent with the docs in numpy and pandas, and with normal writing as well (sentences end in periods). My preference would be to add periods where they're missing instead of removing them.

Reply:
I have a slight preference to remove them as @mauhai has done here.  There is no need to separate sentences here in the title line.

Reply:
Ah, only now seeing @jcrist 's response.  Happy to yield to the numpydoc standard.